CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ option(AKLITE_TAGS "Set to ON to compile with support of aktualizr-lite tag filt
 if(AKLITE_TAGS)
     add_definitions(-DAKLITE_TAGS)
     message(STATUS "Enabling aktualir-lite tag support")
+
+    if (DEFINED ENV{OTA_LITE_TAG})
+        # Take a tag from a spec like:
+        #  https://docs.foundries.io/latest/reference/advanced-tagging.html
+        # and find the first tag name to produce a senible default
+        string(REGEX REPLACE "[,:].*" "" TAG $ENV{OTA_LITE_TAG})
+        message(STATUS "Setting DEFAULT_TAG to " ${TAG})
+        add_definitions(-DDEFAULT_TAG="${TAG}")
+    endif (DEFINED ENV{OTA_LITE_TAG})
```

Comment:
Isn't just better to have it defined via -DDEFAULT_TAG instead and do the env parsing and setting via the OE recipe?

Reply:
probably - i was being lazy.